tests/config/Issue_1813.cfg

```
@@ -4,3 +4,8 @@ indent_namespace               = true
 indent_namespace_single_indent = true
 use_indent_continue_only_once  = true
 indent_cpp_lambda_only_once    = true
+sp_cpp_lambda_paren_brace      = add
+set EXECUTION_CONTEXT DEVICE_LAMBDA_CONTEXT
+set EXECUTION_CONTEXT HOST_DEVICE_LAMBDA_CONTEXT
+set EXECUTION_CONTEXT __device__
+set EXECUTION_CONTEXT __host__ __device__
```

Nitpick: As (sort-of) explained above, this is redundant; the syntax of `set` is `set <TYPE> <TOKEN> [<TOKEN>...]`. Each word is a separate token. So, here, you are making `__host__` a token (desired), and also making `__device__` a token *again*, which you already did in the previous line.

```suggestion
set EXECUTION_CONTEXT __host__ __device__
```